packages/flutter/lib/src/material/tool_bar.dart

```diff
@@ -36,10 +36,13 @@ class ToolBar extends StatelessComponent {
   final TextTheme textTheme;
   final EdgeDims padding;
 
-  ToolBar withPadding(EdgeDims newPadding, { Key fallbackKey }) {
+  ToolBar copyWith({
```

Comment:
We should just bite the bullet and support all the fields. It's weird to have just some supported.